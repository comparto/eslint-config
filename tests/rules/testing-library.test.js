const testingLibrary = require('../../src/rules/jest/testing-library')

test('testing-library rules', () => {
  expect(testingLibrary).toMatchSnapshot()
})
