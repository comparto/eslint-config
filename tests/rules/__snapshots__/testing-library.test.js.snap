// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing-library rules 1`] = `
Object {
  "plugins": Array [
    "testing-library",
  ],
  "rules": Object {
    "testing-library/await-async-query": "error",
    "testing-library/await-async-utils": "error",
    "testing-library/await-fire-event": "off",
    "testing-library/consistent-data-testid": "off",
    "testing-library/no-await-sync-events": "error",
    "testing-library/no-await-sync-query": "error",
    "testing-library/no-debug": "warn",
    "testing-library/no-dom-import": Array [
      "error",
      "react",
    ],
    "testing-library/no-manual-cleanup": "warn",
    "testing-library/no-render-in-setup": "off",
    "testing-library/no-wait-for-empty-callback": "error",
    "testing-library/no-wait-for-snapshot": "warn",
    "testing-library/prefer-explicit-assert": "error",
    "testing-library/prefer-find-by": "error",
    "testing-library/prefer-presence-queries": "error",
    "testing-library/prefer-screen-queries": "off",
    "testing-library/prefer-wait-for": "error",
  },
}
`;
